src/library_pthread.js

```diff
@@ -367,16 +367,19 @@ var LibraryPThread = {
       worker.postMessage({
         'cmd': 'load',
         'handlers': handlers,
+#if expectToReceiveOnModule('mainScriptUrlOrBlob')
         // If the application main .js file was loaded from a Blob, then it is not possible
         // to access the URL of the current script that could be passed to a Web Worker so that
         // it could load up the same file. In that case, developer must either deliver the Blob
         // object in Module['mainScriptUrlOrBlob'], or a URL to it, so that pthread Workers can
         // independently load up the same main application file.
-        'urlOrBlob': Module['mainScriptUrlOrBlob']
+        'urlOrBlob':
+        Module['mainScriptUrlOrBlob']
 #if !EXPORT_ES6
         || _scriptDir
 #endif
```

Comment:
Do we want to set `urlOrBlob` to `_scriptDir` here?

e.g. in the non EXPORT_ES6 case should this be:
```
#if expectToReceiveOnModule('mainScriptUrlOrBlob')
  'urlOrBlob': Module['mainScriptUrlOrBlob'] || _scriptDir,
#else
  'urlOrBlob': _scriptDir,
#endif
```

Actually it looks like we already have a helper function so you should be able to do:

```
 'urlOrBlob': {{{ makeModuleReceiveExpr('mainScriptUrlOrBlob', '_scriptDir') }}}
```